v0/destinations/splitio/data/EventConfig.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "destKey": "eventTypeId",
+    "sourceKeys": [
+      "eventTypeId",
+      "properties.eventTypeId",
+      "event",
+      "name",
+      "traits.eventTypeId",
+      "context.traits.eventTypeId"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": true
+  },
+  {
+    "destKey": "environmentName",
+    "sourceKeys": [
+      "environmentName",
+      "properties.environmentName",
+      "traits.environmentName",
+      "context.traits.environmentName"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": false
+  },
+  {
+    "destKey": "trafficTypeName",
+    "sourceKeys": [
+      "trafficTypeName",
+      "properties.trafficTypeName",
+      "traits.trafficTypeName",
+      "context.traits.trafficTypeName"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "key",
+    "sourceKeys": [
+      "key",
```

Comment:
key is always the user is... see generic mapping and do accordingly same as "userId"

Reply:
did is as userId in generic mapping